bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/model/local/X3HybridG4Data.java

```diff
@@ -129,7 +130,7 @@ public double getFrequencyPhase3() {
 
     @Override
     public double getBatteryVoltage() {
-        return ((double) getFromRawData(39)) / 100;
+        return ((double) ByteUtil.read32BitSigned(getFromRawData(169), getFromRawData(170))) / 100;
```

Comment:
Seems like a complete different position, just a double check this is not a mistake.